scripts/hudson/narayana.sh

```diff
@@ -1050,7 +1050,7 @@ MainClassPatterns="org.jboss.jbossts.qa com.arjuna.ats.arjuna.recovery.RecoveryM
 kill_qa_suite_processes $MainClassPatterns
 
 export MEM_SIZE=1024m
-[ -z ${MAVEN_OPTS+x} ] && export MAVEN_OPTS="-Xms$MEM_SIZE -Xmx$MEM_SIZE"
+[ -z ${MAVEN_OPTS+x} ] && export MAVEN_OPTS="-Xms$MEM_SIZE -Xmx$MEM_SIZE -XX:+IgnoreUnrecognizedVMOptions"
```

Comment:
@mayankkunwar are you able to share which unrecognized options are ignored by this?

Reply:
Okay @tomjenkinson I will add a check for JDK version, if it is less than 17 then only export MAVEN_OPTS